static/src/javascripts/projects/common/modules/article/spacefinder.js

```diff
@@ -60,7 +60,11 @@ define([
 
         // fromBotton:Boolean
         // will reverse the order of slots (this is useful for lazy loaded content)
-        fromBottom: false
+        fromBottom: false,
+
+        // considerScroll:Boolean
+        // will take scroll position into consideration (this is useful for liveblog ads)
+        considerScroll: false
```

Comment:
I removed the `absoluteMinAbove` setting because now we calculate the `top` value based on slot's position relative to the viewport, not absolute value from the top of the page. So we don't need to filter slots again and compare to that `absoluteMinAbove` setting.